UT4MasterServer/Models/Requests/RegisterRequest.cs

```diff
@@ -6,12 +6,14 @@ public class RegisterRequest
 	{
 		private string username = "";
 		private string password = "";
-		
+		private string email = "";
+
+
 		[JsonPropertyName("username")]
 		public string Username { get => username; set => username = value; }
 		[JsonPropertyName("password")]
 		public string Password { get => password; set => password = value; }
 		[JsonPropertyName("email")]
-		public string? Email { get; set; }
+		public string Email { get => email; set => password = value; }

```

Comment:
```suggestion
		public string Email { get => email; set => email = value; }
```
This property is getting `email` but setting `password`